src/Numerics/DGMethods/DGModel.jl

```diff
@@ -1,7 +1,72 @@
 using .NumericalFluxes:
     CentralNumericalFluxHigherOrder, CentralNumericalFluxDivergence
 
-struct DGModel{BL, G, NFND, NFD, GNF, AS, DS, HDS, D, DD, MD}
+"""
+    SpaceDiscretization
+
+Supertype for spatial discretizations.
+
+Must have the following properties:
+
+    - `grid`
+    - `balance_law`
+    - `state_auxiliary`
+"""
+abstract type SpaceDiscretization end
+
+struct DGFVMModel{BL, G, NFND, NFD, GNF, AS, DS, D, DD, MD} <:
```

Comment:
Not a big deal, but would it make sense to call this `DGFVModel` instead?

Reply:
Good idea. Thanks.